### ISSUE TITLE
Forgot Admin password

### ISSUE THREAD

**Issue by DejaVu** [tidusjar/PlexRequests.Net#411]
I know, I know - I thought I made it the same as my Plex.tv details, but nope. Not working.

Is there a way to reset it without deleting and starting again?

**Comment by tidusjar** [comment]
The only way to reset it would be to open the database with something like sqlitebrowser and delete the record from the users table. You will then be able to create a new admin account.

**Comment by stephen304** [comment]
Hey, I'm having a similar issue after upgrading from the old 1-point-something plexrequests.net. I saved my DB, but after enabling password auth in the options (I logged in with my plex user / pass: stephen304) and restarting, I was kicked out and it now rejects my plex credentials. I checked the db and saw an admin account "stephen" so I deleted that, but it still rejects my plex login (my plex user stephen304 is still in the PlexUsers table) and I can login to my local "stephen" account without a password, though "stephen" isn't in users or PlexUsers and when logged in to "stephen", the admin button is gone.

Any advice?
